# Upstream re-located one of their dependencies to another repository. They have
# fixed this in commit: 13922679209b9836554a292a68476aba41d1999c. However there
# have not been any new release  tags cut as yet. As soon as there is a new
# release cut, this can be removed.
diff --git a/pom.xml b/pom.xml
index 25f2d82..7d2e30c 100755
--- a/pom.xml
+++ b/pom.xml
@@ -192,7 +192,7 @@
                             <goal>wget</goal>
                         </goals>
                         <configuration>
-                            <url>https://github.com/riptano/collectd/releases/download/v${collectd.version}/insights-collectd-${collectd.version}.tar.gz</url>
+                            <url>https://github.com/datastax/collectd/releases/download/v${collectd.version}/insights-collectd-${collectd.version}.tar.gz</url>
                             <unpack>true</unpack>
                             <outputDirectory>target</outputDirectory>
                         </configuration>
